docs/reference/experiment-config.txt

```diff
@@ -95,6 +95,12 @@ created with the Determined CLI. It may contain the following fields:
       behavior <https://docs.docker.com/storage/bind-mounts/#configure-bind-propagation>`__
       for replicas of the bind-mount. Defaults to ``rprivate``.
 
+    .. warning::
+
+        When downloading checkpoints from a Determined cluster, we assume the host path
```

Comment:
non-blocking: I think you need to add "a" in front of "shared file system" at the end of the warning. That said, there may be a way to rephrase this entire warning. I probably would have thrown in the word "locally" when we refer to the machine where checkpoints are downloaded.

Reply:
Changed to "When downloading checkpoints in a shared file system, we assume the same shared file system is mounted locally."